Make environment fails on debian during unzipping of sample-data.zip

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

`make environment` fails after installing the required packages on Debian (It will also fail on other Linux distributions as well). I get the following error message:

```
unzip -o sample-data.zip
unzip:  cannot find or open sample-data.zip, sample-data.zip.zip or sample-data.zip.ZIP.
make: *** [Makefile:62: environment] Error 9
```
This is coming from the following line since it tries to unzip a non-existent file because sample-data.zip is saved as sample-data-container.zip by wget command in the previous lines:

https://github.com/microsoft/electionguard-cpp/blob/9e8f3b235f5532aa688ef5ed2023f95a47c1c267/Makefile#L68

### Expected Behavior

`make environment` should succeed.

### Steps To Reproduce

1. Debian Gnu/Linux 11 (although this bug will affect other Linux distributions as well.)
2. Clone the repository
3. run `make environment`

### Environment

```markdown
- OS: Debian Gnu/Linux 11
```


### Anything else?

_No response_

--- a/Makefile
+++ b/Makefile
@@ -65,7 +65,6 @@ endif
 	wget -O cmake/CPM.cmake https://github.com/cpm-cmake/CPM.cmake/releases/download/v0.35.0/CPM.cmake
 	wget -O sample-data-container.zip https://github.com/microsoft/electionguard/releases/download/v0.95.0/sample-data.zip
 	unzip -o sample-data-container.zip
-	unzip -o sample-data.zip
 
 
 build:

